CMake/cdat_modules/vacumm_deps.cmake

```diff
@@ -0,0 +1 @@
+set(vacumm ${vacumm_pkg} ${python_pkg})
```

Comment:
Ok well if that's the case then h5py will likely also need a tweak: https://github.com/UV-CDAT/uvcdat/blob/master/CMake/cdat_modules/h5py_deps.cmake#L1